COMETwebapp/COMETwebapp/Components/Home.razor

```diff
@@ -0,0 +1,13 @@
+﻿<div style="text-align:center">
+    <h1 style="color : @Color">
+        @Title
+    </h1>
+</div>
+
+@code {
+    [Parameter]
+    public string? Title { get; set; }
+
+    [Parameter]
```

Comment:
Please add comment for properties and methods. Maybe discuss with @samatrhea how to comment in Blazor?